omnigibson/objects/controllable_object.py

```diff
@@ -189,6 +188,18 @@ def load(self, scene):
 
         return prim
 
+    def _post_load(self):
+        # Call super first
+        super()._post_load()
+
+        # For controllable objects, we disable gravity of all links that are not the base link. This
+        # is because we cannot accurately apply gravity compensation in the absence of a working
+        # generalized gravity force computation. This may have some side effects on the measured
+        # torque on each of these links, but it provides a greatly improved joint control behavior.
+        for link in self._links.values():
+            if link is not self.root_link:
```

Comment:
this is quite elegant, but may be problematic depending on the robot. For example, isn't Tiago's root link technically x_base_footprint? And a robot's root link may not actually be the highest mass link

A very hacky, but perhaps more accurate heuristic could be to only select the largest mass link to be gravity enabled.

Reply:
I have implemented our discussesd solution.